tests/suites/test_suite_ssl.function

```diff
@@ -6247,6 +6247,9 @@ void ssl_ecjpake_set_password( int use_opaque_arg )
 
         ECJPAKE_TEST_SET_PASSWORD( MBEDTLS_ERR_SSL_HW_ACCEL_FAILED );
 
+        /* check that the opaque key is still valid after failure */
+        TEST_ASSERT( ! mbedtls_svc_key_id_is_null( pwd_slot ) );
```

Comment:
I erroneously thought that a call to `psa_destroy_key` would also destroy the key-id, so instead of getting the attributes I preferred `mbedtls_svc_key_id_is_null`. However, if I understood correctly, it clears the slot but not the ID itself, so in that sense my original hypothesis was not correct.